README.md

```diff
@@ -32,14 +32,17 @@ Diese MCU hat ein sehr kleines Display, dafür aber zusätzlich zu WiFi noch LoR
 Es wird in die kürzeren Buchsenleisten gesteckt.
 Für die Arduino-IDE muss als Board **Heltec Wireless Stick** eingestellt werden.
 
+Um die Heltec-Boards in der Arduino IDE auswählen zu können, muss "https://docs.heltec.cn/download/package_heltec_esp32_index.json" in den Preferences unter Additional Boards Manager URLs hinzugefügt werden. Danach können unter Tools->Board->Boards Manager die Heltec Boards (Name "Heltec ESP32...")
```

Comment:
https://raw.githubusercontent.com/espressif/arduino-esp32/gh-pages/package_esp32_dev_index.json

ich hab da diese URL von espressif (esp32-hersteller) eingetragen und bekam dann auch heltec boards zur auswahl - aber eben nicht nur die.

Reply:
Genau die verwende ich auch. Sollen wir da dann einen Verweis  in das Readme setzen?
So etwa: um mit der Arduino-IDE ESP32 zu üb ersetzten, müssen die ESP32 Definition eingebunden werden ( Anleitung siehe z.B.: https://www.heise.de/ct/artikel/Arduino-IDE-installieren-und-fit-machen-fuer-ESP8266-und-ESP32-4130814.html).

Reply:
Ich wuerde die der .cn url bevorzugen, ansonsten kann man das so wie o.g. lassen.

Reply:
Also gut, dann merge ich das, wie dc6jgk vorgeschlagen hat

Reply:
Hmm, sorry, hab mich da missverstaendlich ausgedrueckt.

"die" bezog sich auf die espressif-url - und "die" wuerde ich der in China (heltec.cn) bevorzugen.

Schlechte Erfahrungen mit .cn und ausserdem sollten bei espressif mehr und auch andere Hersteller drin sein.

Reply:
OK, so she ich das eher auch. Ich ändere das direkt online, ohne PR.

Reply:
Done